lib/configgin.rb

```diff
@@ -178,7 +178,7 @@ def expected_annotations(job_configs, job_digests)
       base_config = JSON.parse(File.read(job_config['base']))
       base_config.fetch('consumed_by', {}).values.each do |consumer_jobs|
         consumer_jobs.each do |consumer_job|
-          digest_key = "skiff-in-props-#{instance_group}-#{job_name}"
+          digest_key = "skiff-#{ENV['CONFIGGIN_VERSION_TAG']}-#{instance_group}-#{job_name}"
```

Comment:
@jandubois In our initial discussion I got the impression that the helm chart version would be used here somehow. Now I am seeing the configgin version ?! How does that work when the user updates from a chart V to chart V+1 which both have the same configgin version C ?

Reply:
`CONFIGGIN_VERSION_TAG` *is* the helm chart version:

```
        - name: "CONFIGGIN_VERSION_TAG"
          value: {{ .Chart.Version }}-{{ .Values.kube.secrets_generation_counter }}
```

The `CONFIGGIN_` prefix exist only to designate the consumer of this value, which is auto-generated by fissile.

Reply:
Ah! Should have checked further instead of just going by the name.